.gitignore

```diff
@@ -1,24 +1,3 @@
 node_modules
 .env
-
```

Comment:
question: does having multiple .gitignore files in one repository work? I thought only the one that is in the root folder is considered by Git.

Reply:
Yeah it works, and was suggested by @zoruka in the previous PR. 
![image](https://user-images.githubusercontent.com/18757934/217019393-fb4c1430-5529-4e7b-afe1-2bf87fb10781.png)

Looked it up and it says that so we just need to be careful of how they are applied together but it works. That's why those files didn't get committed either.